src/utils/aria.ts

```diff
@@ -5,6 +5,8 @@ export const ariaLabelOrHidden = (ariaLabel?: string) =>
     ? { 'aria-label': ariaLabel, role: 'img' }
     : { 'aria-hidden': true };
 
+export type AriaLiveMode = 'assertive' | 'polite' | 'off';
+
```

Comment:
Nitpicking here, but as this is not an util it could be located to `src/core/Form/types.ts`

_On the other end, filename `types.ts` is also somehow misleading when the `interface StatusTextCommonProps` was added. But interface and type declarations seems to close in TypeScript so maybe we can live with this._

Reply:
Moved the type to types.ts